Can't use a config without uart

when trying to use a config without uart the compiler fails with the message during elaboration.

> [error] Caused by: java.lang.IllegalArgumentException: requirement failed: Key PeripheryUARTKey is not defined in Parameters

it seems that in recent versions the chip must have a uart periphery. without it the compiler would fail.

--- a/generators/chipyard/src/main/scala/ConfigFragments.scala
+++ b/generators/chipyard/src/main/scala/ConfigFragments.scala
@@ -53,7 +53,11 @@ class WithUART extends Config((site, here, up) => {
 })
 
 class WithNoGPIO extends Config((site, here, up) => {
-  case PeripheryGPIOKey => Seq()
+  case PeripheryGPIOKey => Nil
+})
+
+class WithNoUART extends Config((site, here, up) => {
+  case PeripheryUARTKey => Nil
 })
 
 class WithL2TLBs(entries: Int) extends Config((site, here, up) => {

